sqlx-postgres/src/options/mod.rs

```diff
@@ -492,6 +496,20 @@ impl PgConnectOptions {
         self
     }
 
+    /// Sets the path to the file used to store passwords. Defaults to None.
+    ///
```

Comment:
This should mention that `.apply_pgpass()` needs to be called for this to take effect.

I think maybe the `.pgpass` handling needs to be reworked entirely, because people seem to be continually surprised that `new()` tries to read it.